openfe/tests/setup/test_lomap_atommapper.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+import openfe
+from openfe.setup.lomap_mapper import LomapAtomMapper
+
+
+def test_simple(lomap_basic_test_files):
+    # basic sanity check on the AtomMapper
+    mol1 = lomap_basic_test_files['methylcyclohexane']
+    mol2 = lomap_basic_test_files['toluene']
+
+    mapper = LomapAtomMapper()
+
+    mapping_gen = mapper.suggest_mappings(mol1, mol2)
+
+    mapping = next(mapping_gen)
+    assert isinstance(mapping, openfe.setup.AtomMapping)
+    # methylcyclohexane to toluene is a 1:1 mapping between all atoms
+    # so 7 values should be present
+    assert len(mapping.mol1_to_mol2) == 7
```

Comment:
As discussed privately - if you can check the contents themselves that'd be great.

Possibly the only other thing is maybe there should be more cases being checked here, but it's up to you. Something like a "no atoms map" case could be good to test.

Reply:
If we're waiting on fixing lomap - shall we raise an issue for expanding the tests at a later point?

Reply:
lomap regression tests started, so this isn't a here problem